server/interpreter.js

```diff
@@ -1217,6 +1303,48 @@ Interpreter.prototype.initError = function(scope) {
 };
 
 /**
+ * Initialize the thread system API
+ * @param {!Interpreter.Scope} scope Global scope.
+ */
+Interpreter.prototype.initThreads = function(scope) {
+  var intrp = this;
+  this.addVariableToScope(scope, 'suspend', this.createNativeFunction(
+      function(delay) {
+        delay = Number(delay || 0);
```

Comment:
Good catch.  I can only assume I was confusedly thinking that `(NaN || 0) === NaN`, but of course that is not true (no pun in tended).